configuration/errors.go

```diff
@@ -0,0 +1,13 @@
+package configuration
```

Comment:
Let's not create a new file just yet, I think this error could live in `configuration/parser.go` instead